docs/troubleshooting/podman-wsl-not-found.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Container runtime 'podman' could not be found in WSL
+description: Learn how to troubleshoot the error "Container runtime 'podman' could not be found" when using Podman in Windows Subsystem for Linux (WSL).
+ms.date: 08/04/2025
+---
```

Comment:
Can this article be written according to the troubleshooting template at https://review.learn.microsoft.com/en-us/help/patterns/level4/global-problem-resolution-template?branch=main